### ISSUE TITLE
Not compatible with pedestal 0.4.0

### ISSUE THREAD

**Issue by cch1** [propan/geheimtur#5]
```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at clojure.main.<clinit>(main.java:20)
Caused by: java.lang.IllegalAccessError: definterceptorfn does not exist, compiling:(geheimtur/interceptor.clj:1:1)
```

**Comment by propan** [comment]
Should be fixed in 0.1.4

**Comment by cch1** [comment]
That was quick.  Thanks.

-Chris
